use-cases/virgo/synthetic-data-gen/data_generation_hdf5.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#SBATCH --account=intertwin
+#SBATCH --output=array-job/job_%a.out
+#SBATCH --error=array-job/job_%a.err
+#SBATCH --time=00:07:00
+#SBATCH --mem-per-cpu=1G
+#SBATCH --partition=develbooster
+#SBATCH --array=1-75
+#SBATCH --job-name=generate_virgo_data
+#SBATCH --cpus-per-task=26
+
+# Load required modules
+ml Stages/2024 GCC OpenMPI CUDA/12 MPI-settings/CUDA Python HDF5 PnetCDF libaio mpi4py
+
+# Activate Python virtual environment
+source ../../envAI_hdfml/bin/activate
```

Comment:
Maybe change this to the jewels environment (../../envAI_juwels/bin/activate). This will come up a few more times probably

Reply:
I agree that this should be changed. I have it this way right now since I haven't changed the name of my venv, but I think I might have a better idea about how to handle this naming thing anyway. See Mattermost.